projects/libgd/project.yaml

```diff
@@ -0,0 +1,4 @@
+homepage: "https://libgd.org"
+primary_contact: "<primary_contact_email>"
```

Comment:
the primary contact is https://github.com/libgd/libgd/issues

but we also have `security@libgd.org` which might fit the bill here.

Reply:
@vapier Is that email address a Google account? You'll want to add at least one so that you can get access to the private issue tracker and web UI for looking at crashes/stats.

Reply:
As @titanous reminds me (thanks!), I think it's helpful to use personal Google accounts because they're used for authentication to the [ClusterFuzz dashboard](https://github.com/google/oss-fuzz/blob/master/docs/faq.md#why-do-you-require-a-google-account-for-authentication).

Reply:
the GD project has no official Google account for contact.  i personally have a `vapier@gmail.com` account.  i don't know about Pierre/Christoph/Ondřej.

Reply:
I have a Google account, but no Google mail address. I could obtain one, if necessary.

Reply:
Any Google account is fine, it doesn't need to be Gmail.

Reply:
Thanks, @titanous ! How do I get access to https://bugs.chromium.org/p/oss-fuzz/issues/ via `cmbecker69@gmx.de`?

Reply:
@cmb69 send a PR to add yourself to the auto_ccs list in this file.

Reply:
Thanks. [Done](https://github.com/google/oss-fuzz/pull/1117)